spec/controllers/account/notifications_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Account::NotificationsController, type: :controller do
+  let!(:user) { create(:user, first_name: 'Bob') }
+  let!(:notification) { create(:notification, notification_type: Notification::NOTIFICATION_TYPES.sample, notificable: user, user_id: user.id) }
+
+  describe 'mark_all_as_read' do
+    it 'marks all notifications as read' do
+      sign_in user
```

Comment:
separate the example logical flows
i've sent u several links to read about code style